libmproxy/models/http.py

```diff
@@ -11,9 +11,10 @@
 from .. import version, stateobject
 from .flow import Flow
 
+from collections import OrderedDict
 
 class MessageMixin(stateobject.StateObject):
-    _stateobject_attributes = dict(
+    _stateobject_attributes = OrderedDict(
```

Comment:
kk. Let's add a quick comment explaining that so we can fix that nicely at a later point.